api_tests/performance/locustfile.py

```diff
@@ -0,0 +1,21 @@
+import os
+from locust import HttpUser, TaskSet, task, between
+
+class IdentityServiceUser(HttpUser):
+    wait_time = between(2, 5)
+
+    def on_start(self):
+        self.identity_proxy = self._identity_proxy_name()
+        self.client_id = os.environ["CLIENT_ID"]
+        self.callback_url = os.environ["CALLBACK_URL"]
+
+    def _identity_proxy_name(self):
+        try: 
+            namespace = os.environ["NAMESPACE"]
+            return f"oauth2-{namespace}"
+        except:
```

Comment:
Ideally should be more specific with the exception (`except KeyError`)... alternatively use the `.get('key', 'default-value')` dictionary method